js/components/portal-dashboard/student-navigator.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import ArrowLeftIcon from "../../../img/svg-icons/arrow-left-icon.svg";
+
+import css from "../../../css/portal-dashboard/student-navigator.less";
+import { getFormattedStudentName } from "../../util/student-utils";
+
+
+interface IProps {
+  students: any;
```

Comment:
Probably typed as Map<any, any>

Reply:
students is types as any in portal-dashboard-app and passed around all the different components as that typed. Maybe can be refactored to the correct type later?

Reply:
Yeah, I think we built some of those components early before Sam worked out what the types were so we temporarily slotted in any.  If it causes grief to make this Map<any, any>, then I'll make a story to fix these throughout components later.

Reply:
Yeah, that looks to cause trouble.  Let's leave it as any for now.  I'll make a note to fix it in the future.